flake.nix

```diff
@@ -28,10 +28,11 @@
       # Complete repository: larger and more varied collection.
       # Naturally, means longer build times.
       full = wallpkgs;
-      wallpkgs = callWallpaper null;
```

Comment:
`callWallpaper` is designed to handle null, not an empty string.

Is there a good reason why you're making this change?

Reply:
it causes nix flake show and the direnv file to have a panic attack

Reply:
![image](https://github.com/user-attachments/assets/22d4e25a-24b9-49fd-a445-ef9639c00789)

Reply:
I see the problem actually. Setting it to `""` doesn't fix the problem. I'll handle it better in the derivation.

Reply:
found and fixed it myself.